examples/iw_vae.py

```diff
@@ -190,12 +175,14 @@ def latent_gaussian_x_bernoulli(z, z_mu, z_log_var, x_mu, x, eq_samples, iw_samp
 
     # dimshuffle x since we need to broadcast it when calculationg the binary
     # cross-entropy
-    x = x.dimshuffle((0,'x','x',1))
+    x = x.dimshuffle(0,'x','x',1) # x: (batch_size, nsamples, ivae_samples, num_latent)
```

Comment:
comment should be changed to (batch_size, nsamples, ivae_samples, num_latent)  should be changed to 
(batch_size, eq_samples, iw_samples, num_latent) i think